test/basicShapePolygonTest.js

```diff
@@ -0,0 +1,38 @@
+/* global suite test assert internalScope */
+'use strict';
+
+(function () {
+  suite('offsetPath', function () {
+    test('polygon', function () {
+      var offsetPathParse = internalScope.offsetPathParse;
+      var assertTransformInterpolation = internalScope.assertTransformInterpolation;
+
+      var result = offsetPathParse('polygon(150px 200px, 250px 150px, 200px -150px)');
```

Comment:
I suspect when we make my m x y change, you will need to start with polygon(0px 0px, ...) to get the current parse result and transform results.